src/bundle/Resources/views/themes/admin/matrix_fieldtype/content_edit_form_fields.html.twig

```diff
@@ -7,12 +7,16 @@
             {{ form_label(form) }}
         </div>
         <div>
-            <button type="button" class="btn btn-primary ez-btn--add-matrix-entry">
+            <button type="button" 
+                    class="btn btn-primary ez-btn--add-matrix-entry" 
+                    title="{{ 'ezmatrix.add'|trans({}, 'matrix_fieldtype')|desc('Add') }}">
```

Comment:
It's better to use `{% trans_default_domain 'matrix_fieldtype' %}` at the top of a file. Thanks to that all translations in this file will be in the same domain.